installers/rpm-code/codes/mantid.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+mantid_main() {
+    codes_yum_dependencies \
+        eigen3-devel \
+        OCE-devel \
+        jemalloc-devel \
+        jsoncpp-devel \
+        muParser-devel \
+        poco-devel
+    codes_dependencies common boost ipykernel
+    install_pip_install pre-commit pylint sphinx_bootstrap_theme
+    mantid_nexus_install
+    # Mantid installs a great deal and requires its own specific environment variables to run right
+    # (ex. see the mantidpython script it creates). So, install it in its own subdirectory so it
+    # doesn't interfere with other libs.
+    local p="${codes_dir[lib]}/mantid"
+    mantid_install "$p"
+    mantid_ipykernel $(mantid_install_rsmantid "$p")
+}
+
+mantid_install() {
+    local install_prefix="$1"
+    codes_download mantidproject/mantid
+    mantid_patch_eigen_cmake
+    codes_cmake \
+        -DBOOST_ROOT="${codes_dir[prefix]}" \
+        -DCMAKE_INSTALL_PREFIX="$install_prefix" \
+        -DENABLE_WORKBENCH=OFF \
+        -DMANTID_FRAMEWORK_LIB=BUILD \
+        -DMANTID_QT_LIB=OFF \
+        -DNEXUS_CPP_LIBRARIES="${codes_dir[lib]}/libNeXusCPP.so" \
+        -DNEXUS_C_LIBRARIES="${codes_dir[lib]}/libNeXus.so" \
+        -DNEXUS_INCLUDE_DIR="${codes_dir[include]}/nexus/" \
+        -G'Unix Makefiles'
+    codes_cmake_build
+    codes_make_install
+}
+
+mantid_install_rsmantid() {
+    local s=rsmantid
+    codes_download_module_file "$s.sh"
+    local l="$1/bin/mantidpython"
```

Comment:
I wouldn't use a temporary for this. It doesn't improve readability. Just echo $MANTID_PYTHON_SCRIPT_INSTALL_LOCATION (below)

Reply:
Can you show me what you mean? `$MANTID_PYTHON_SCRIPT_INSTALL_LOCATION` is just set on the perl command. Should I define it as a local variable instead of "$l"?